bounce/db/club.py

```diff
@@ -17,7 +16,8 @@
 MAX_SIZE = 20
 
 # Defining a enum type for role allocation
-role = ENUM('President', 'Admin', 'Member', name='role')
+# TODO: Do we need this?  What's the reason we have this in db/club.py?
```

Comment:
We can probably import role as well? This is just so we can use it in clubs.

Reply:
Ok.  That works.  I think importing would be good.